ios/AirGoogleMaps/AIRGoogleMapMarkerManager.m

```diff
@@ -23,6 +23,8 @@ - (UIView *)view
 //  tapGestureRecognizer.cancelsTouchesInView = NO;
 //  [marker addGestureRecognizer:tapGestureRecognizer];
   marker.bridge = self.bridge;
+  marker.isAccessibilityElement = "YES";
+  marker.accessibilityElementsHidden = "NO";
```

Comment:
1. Are these supposed to be strings?
2. It seems like we shouldn't set these here, but rather set them when/if people set the `testID` prop. (or other accessibility props if we add them, which would be great if we did)